src/app/grants/page.tsx

```diff
@@ -171,16 +172,25 @@ export default function Grants() {
         }}
       />
 
-      <section>
-        <Heading tag="h2" variant="xl">
-          How to get involved in the Filecoin community
+      <section className="m-auto space-y-6 text-left sm:text-center">
+        <Heading tag="h2" variant="3xl">
+          Ready to Apply?
         </Heading>
-        <p>
-          The Filecoin community includes thousands of developers,
-          technologists, users, and enthusiasts all over the world.
+        <p className="max-w-[60ch]">
+          Please visit our{' '}
+          <TextLink href={FILECOIN_URLS.grants.github}>Github repo</TextLink> to
+          learn more about the proposal process, review process, timeline
+          guidance, and more. For other questions, email{' '}
+          <TextLink href={FILECOIN_URLS.grants.email}>grants@fil.org</TextLink>{' '}
+          or join our monthly office hours!
         </p>
-        <TextLink href={FILECOIN_URLS.social.slack.href}>Learn More</TextLink>
-        <GetInvolvedList />
+        <Button
+          className="sm:self-start"
+          variant="primary"
+          href={FILECOIN_URLS.grants.email}
+        >
+          Apply Now
+        </Button>
       </section>
```

Comment:
@barbaraperic Can you please have a look at the Figma designs, and confirm that we'll need to this component in a few pages? If so, let's create a component for it!

Reply:
You're right, there is a similar section. I've extracted the section styles and heading, left paragraph and button as children as they can vary with icons and inner text links.

![Screenshot 2024-04-22 at 09 51 18](https://github.com/FilecoinFoundationWeb/filecoin-foundation/assets/50910606/b68826ad-aecc-4ade-8deb-f66a4e7fda40)

Reply:
I understand the thinking. Still, we should try and move the text and button inside of the component. Not unlike we do for the optional `cta` in  `Card` (please note that `Card` needs some refactoring so asses yourself what can be reused!)

```
        {cta && (
          <CustomLink
            href={cta.href}
            aria-label={cta.ariaLabel}
            className="absolute inset-0 rounded-lg focus:outline-2 focus:outline-brand-100"
          >
            <span className="absolute bottom-4 left-4 inline-flex items-center gap-2 text-brand-300">
              {cta.icon || <BookOpen size={24} aria-hidden={true} />}
              <span>{cta.text || 'Learn More'}</span>
            </span>
          </CustomLink>
        )}
```        

As for the up arrow icon with outgoing links we should add that as the permanent icon under `Button` for external links.

Reply:
Also - You don't need to take all options into account in this PR - Make the component work for this PR - and then we can get to the variants once we need to.